docs/src/pages/guides/routing/routing.md

```diff
@@ -63,6 +63,10 @@ You can learn more about this prop in the [composition guide](/guides/compositio
 Here are a few demos with [react-router](https://github.com/ReactTraining/react-router).
 You can apply the same strategy with all the components: BottomNavigation, Card, etc.
 
+**Note**: The `ButtonBase` adds this attribute: `role="button"` when it identifies the intent to render a button without a native `<button>` element.
+This can create issues when rendering a link. If you are not using one of the `href`, `to`, or `componnet="a"` props you need to set `role={undefined}`.
+For instance:
```

Comment:
@oliviertassinari just want to get this to your notice a small miss spell is there `componnet="a"`